src/scenes/ObservationsRouterV2/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Route, Routes } from 'react-router';
+
+import ObservationListView from './ListView';
+
+const ObservationsRouter = (): JSX.Element => {
+  return (
```

Comment:
I think someone was talking recently about naming components the same as their files. Would it be worth renaming this to `ObservationsRouterV2`? I know that it will likely replace the other eventually, just curious since duplicate names like this can confuse IDE's like IntelliJ